packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -63,7 +74,38 @@ class Chat extends Widget<Properties> {
   _renderMessageBox(): void {
     const $messageBox = $('<div>').appendTo(this.element());
 
-    this._messageBox = this._createComponent($messageBox, MessageBox, {});
+    const configuration: MessageBoxProperties = {
+      onMessageSend: (text) => {
+        this._messageSendHandler(text);
+      },
+    };
+
+    this._messageBox = this._createComponent($messageBox, MessageBox, configuration);
+  }
+
+  _createMessageSendAction(): void {
+    this._messageSendAction = this._createActionByOption(
+      'onMessageSend',
+      { excludeValidators: ['disabled', 'readOnly'] },
+    );
+  }
+
+  // eslint-disable-next-line max-len
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+  _messageSendHandler(action: any): void {
```

Comment:
```suggestion
  _messageSendHandler(e: MessageSendEvent): void {
```

Reply:
After this don't remove delete eslint comments

Reply:
[cd52ce11e169517258f5b51a5b14ac75315205da]